### PR TITLE
Squashed commit of the following:

### PR DESCRIPTION
**Pull request by garris**
commit 46147e1889a25b190f16daade9df2be2e4eb4708
Author: Garris garris@me.com
Date:   Fri Aug 29 18:42:58 2014 -0700

```
added comments to jshint config

commented laxcomma & smarttabs options in jshint config
```

commit 2645c0e60112fcb4c666c747af1b4c11d9aa6501
Author: Garris garris@me.com
Date:   Fri Aug 29 18:34:31 2014 -0700

```
fixed vendor css compile issue

Added `build_vedndorcss` block and added block to copy vendor_files.css to bin on compile job.
```

commit 7f70c9cac39b89719a1fb1f1d32734e3d8801929
Author: Garris garris@me.com
Date:   Thu Aug 28 16:52:34 2014 -0700

```
fixed vendor CSS deploy issue

CSS files listed in /build.config.js > vendor_files.css are now correctly deployed to /build/vendor/[dirName...] as expected.
```

### CODE REVIEW

**Comment by garris** [comment]
Per @Merott -- squashed fix here.

File: Gruntfile.js [hunk 1]
```diff
@@ -278,6 +294,8 @@ module.exports = function ( grunt ) {
         'Gruntfile.js'
       ],
       options: {
+        //smarttabs:true,
+        //laxcomma: true,
```
Comment (Merott) [inline Gruntfile.js:298]: Please remove these two lines, as they're not related to the fix.

**Comment by Merott** [comment]
Thank @garris 

Just a couple of things:
- Please change the first line of commit to follow the [AngularJS guidelines](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format) for Git commit message format.
- Please remove unrelated changes under the `jshint` configuration.
- Squash the changes again.

Thank you for your help.

**Comment by garris** [comment]
Great advice. Will do.

**Comment by garris** [comment]
closing this pull request. creating new one per advice.

**Pull request closed without merging**